subsys/audio_module/Kconfig

```diff
@@ -17,14 +17,17 @@ config AUDIO_MODULE_TEST
 
 config AUDIO_MODULE_NAME_SIZE
 	int "Maximum size for module naming in characters"
+	depends on AUDIO_MODULE
 	default 20
 
 #----------------------------------------------------------------------------#
 menu "Log levels"
```

Comment:
why does this need it's own menu? Which adds to confusion because it will always be displayed even when the module is disabled

Reply:
It is probably not needed. Keeping it here for now